From 5e7ac89a59fb519555c591e7e587f28da9cf942a Mon Sep 17 00:00:00 2001
From: David Marble <drmarble1@gmail.com>
Date: Tue, 19 Nov 2013 16:06:23 +0100
Subject: [PATCH] We need our own fsck for tenderloin This breaks recovery,but
 then, it doesn't work for us anyway, we use a custom

---
 Android.mk      | 6 +++---
 recovery_cmds.h | 2 --
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/Android.mk b/Android.mk
index 9b66f54..ecf2071 100644
--- a/Android.mk
+++ b/Android.mk
@@ -82,7 +82,7 @@ $(foreach board_define,$(BOARD_RECOVERY_DEFINES), \
 LOCAL_STATIC_LIBRARIES :=
 
 LOCAL_CFLAGS += -DUSE_EXT4 -DMINIVOLD
-LOCAL_C_INCLUDES += system/extras/ext4_utils system/core/fs_mgr/include external/fsck_msdos
+LOCAL_C_INCLUDES += system/extras/ext4_utils system/core/fs_mgr/include
 LOCAL_C_INCLUDES += system/vold
 
 LOCAL_STATIC_LIBRARIES += libext4_utils_static libz libsparse_static
@@ -100,7 +100,7 @@ else
   LOCAL_SRC_FILES += $(BOARD_CUSTOM_RECOVERY_KEYMAPPING)
 endif
 
-LOCAL_STATIC_LIBRARIES += libvoldclient libsdcard libminipigz libfsck_msdos
+LOCAL_STATIC_LIBRARIES += libvoldclient libsdcard libminipigz
 LOCAL_STATIC_LIBRARIES += libmake_ext4fs libext4_utils_static libz libsparse_static
 
 ifeq ($(TARGET_USERIMAGES_USE_F2FS), true)
@@ -126,7 +126,7 @@ LOCAL_STATIC_LIBRARIES += libselinux
 
 include $(BUILD_EXECUTABLE)
 
-RECOVERY_LINKS := bu make_ext4fs edify busybox flash_image dump_image mkyaffs2image unyaffs erase_image nandroid reboot volume setprop getprop start stop dedupe minizip setup_adbd fsck_msdos newfs_msdos vdc sdcard pigz
+RECOVERY_LINKS := bu make_ext4fs edify busybox flash_image dump_image mkyaffs2image unyaffs erase_image nandroid reboot volume setprop getprop start stop dedupe minizip setup_adbd newfs_msdos vdc sdcard pigz
 
 ifeq ($(TARGET_USERIMAGES_USE_F2FS), true)
 RECOVERY_LINKS += mkfs.f2fs fsck.f2fs fibmap.f2fs
diff --git a/recovery_cmds.h b/recovery_cmds.h
index 3d1d1fa..d8d0667 100644
--- a/recovery_cmds.h
+++ b/recovery_cmds.h
@@ -36,7 +36,6 @@ extern int reboot_main(int argc, char **argv);
 extern int poweroff_main(int argc, char **argv);
 extern int setprop_main(int argc, char **argv);
 extern int getprop_main(int argc, char **argv);
-extern int fsck_msdos_main(int argc, char **argv);
 extern int newfs_msdos_main(int argc, char **argv);
 extern int vdc_main(int argc, char **argv);
 extern int pigz_main(int argc, char **argv);
@@ -71,7 +70,6 @@ static const struct recovery_cmd recovery_cmds[] = {
     { "poweroff",       reboot_main },
     { "setprop",        setprop_main },
     { "getprop",        getprop_main },
-    { "fsck_msdos",     fsck_msdos_main },
     { "newfs_msdos",    newfs_msdos_main },
     { "vdc",            vdc_main },
     { "pigz",           pigz_main },
-- 
1.8.1.2

